src/Ng/lightning-template.txt

```diff
@@ -2,15 +2,15 @@
 
 :Top
 
-title [cursor]
+title [cursorFile]
 
 @echo starting
 
-start /w ng.exe lightning -command strike -indexFile [index] -cursorFile [cursor] -contentBaseAddress [contentbaseaddress] -storageAccount [storageaccount] -storageContainer [storagecontainer] -storageBaseAddress [storagebaseaddress] -compress [compress] -verbose true
+start /w ng.exe lightning -command strike -indexFile "[indexFile]" -cursorFile "[cursorFile]" -contentBaseAddress "[contentBaseAddress]" -verbose "[verbose]" -storageBaseAddress "[storageBaseAddress]" -storageAccountName "[storageAccountName]" -storageKeyValue "[storageKeyValue]" -storageContainer "[storageContainer]" -useCompressedStorage "[useCompressedStorage]" -compressedStorageBaseAddress "[compressedStorageBaseAddress]" -compressedStorageAccountName "[compressedStorageAccountName]" -compressedStorageKeyValue "[compressedStorageKeyValue]" -compressedStorageContainer "[compressedStorageContainer]" -useSemVer2Storage "[useSemVer2Storage]" -semVer2StorageBaseAddress "[semVer2StorageBaseAddress]" -semVer2StorageAccountName "[semVer2StorageAccountName]" -semVer2StorageKeyValue "[semVer2StorageKeyValue]" -semVer2StorageContainer "[semVer2StorageContainer]"
```

Comment:
It seems a little dangerous to me that whenever we change the parameters of the job, the person needs to know to update this script as well. It's probably outside the scope of this PR, but it'd be nice if the job just copied over all of the commands that aren't `prepare` specific.